changelog.md

```diff
@@ -1,3 +1,9 @@
+5.1.0
+========
+## API additions to enable support for managed identities.
+- Created a new [ManagedIdentityOptions class](https://github.com/AzureAD/microsoft-identity-abstractions-for-dotnet/blob/main/src/Microsoft.Identity.Abstractions/TokenAcquisition/ManagedIdentityOptions.cs).
+- Added a 'ManagedIdentity' property to the [AcquireTokenOptions class](https://github.com/AzureAD/microsoft-identity-abstractions-for-dotnet/blob/main/src/Microsoft.Identity.Abstractions/TokenAcquisition/AcquireTokenOptions.cs).

```

Comment:
usually we just say "## New feature" and add a link to the GitHub issue, but this is fine...can you add the GitHub issue link as well?